cgotorch/cgotorch.h

```diff
@@ -10,7 +10,20 @@ extern "C" {
 
 typedef void *Tensor;
 typedef void *Optimizer;
-char *RandN(int rows, int cols, int require_grad, Tensor *result);
+// torch.randn
+char *RandN(int64_t *size, int64_t length, int64_t require_grad,
```

Comment:
Should we make the return type from `char *` to `const char *`?

Reply:
Yes, we do not modify the return error message. `const char*` is more accurate.